func_sim/func_instr/func_instr.cpp

```diff
@@ -269,12 +271,15 @@ void FuncInstr::initI()
 
 void FuncInstr::initJ()
 {
-    dst   = REG_NUM_ZERO;
     v_imm = instr.asJ.imm;
 
     std::ostringstream oss;
     oss << isaTable[isaNum].name << " "
-        << std::hex << "0x" <<instr.asJ.imm;
+        << std::hex << "0x" << v_imm;
+
+    if ( operation == OUT_J_JUMP_LINK)
+        dst = REG_NUM_RA;
```

Comment:
It's not kept explicitly in, for example, `case OUT_I_BRANCH`, as well. But that's not a problem, as it is set already before calling initFormat()